content/02.introduction.md

```diff
@@ -1,12 +1,26 @@
 ## Introduction
 
-1. Talk about knowledge graphs
-  1. Define what they are
-  2. Provide a diagram to orient the reader
-2. Mention how these graphs are constructed 
-3. Mention some applications these graphs can be used for
-	1. Biomedical setting
-	2. Social Media 
-	3. Question and answering systems 
-	4. more if it comes to mind
-4. Conclude section by mentioning that this review dives deeper into approaches used for construction and applying these graphs to biomedical applications
+Knowledge graphs are a practical resource for a multitude of real world applications.
+This resource utilizes graph theory to represent knowledge in the form of a graph.
+Nodes represent entites and edges represent relational information between two entities (example shown in Figure {@fig:hetionet_schema}).
+
+Typically, knowledge graphs are constructed via manually curated databases, but there are approaches that utilize natural language processing techniques for construction.
+For example, one can construct a text mining system to detect sentneces that indicate a protein-protein interactions [@doi:10.1016/j.knosys.2018.11.020].
+After the sentences have been identified, they are incorporated as evidence for establishing an edge in a knowledge graph.
+
+Besides construction, knowledge graphs have been used in biomedical and non-biomedical applications (e.g. disease gene prioritization, social media mining, interpreting natural language, etc).
```

Comment:
I would probably start with their utility. It looks like many of these things would be helpful in the first paragraph.